src/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
it's not a bad solution, but it would be better to use font-face or google fonts